torch_em/model/probabilistic_unet.py

```diff
@@ -0,0 +1,445 @@
+# This code is based on the original TensorFlow implementation: https://github.com/SimonKohl/probabilistic_unet
+# The below implementation is from: https://github.com/stefanknegt/Probabilistic-Unet-Pytorch
+
+import numpy as np
+
+import torch
+import torch.nn as nn
+from torch.distributions import Normal, Independent, kl
+
+from torch_em.model import UNet2d
+from torch_em.loss.dice import DiceLossWithLogits
+
+device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
```

Comment:
Don't define this globally. Instead pass this as an argument to the functions or arguments that need it.